mappings/net/minecraft/entity/ai/AiGoalTrackTarget.mapping

```diff
@@ -0,0 +1,32 @@
+CLASS xu net/minecraft/entity/ai/AiGoalTrackTarget
+	FIELD a checkCanNavigate Z
+	FIELD b canNavigateFlag I
```

Comment:
I agree with it being "flag" if it's just one multi-state flag as opposed to mutliple flags